index.bs

```diff
@@ -144,6 +147,8 @@ The Accept-CH cache can effectively act as an alternative cookie store,
 since sites can use each of the hints as a bit set on the client, and that information will be
 communicated to them on every request. As such, a user agent MUST evict that
 cache whenever the user clears their cookies or when session cookies expire.
+A user agent also MUST clear the Accept-CH according to the rules of the
+[=Clear-Site-Data Header=].
```

Comment:
Would be nice to send a PR to export the dfn (https://github.com/w3c/webappsec-clear-site-data/blob/78a1b5b6ab4f3070b9422b80b81e62bf933da72f/index.src.html#L247) - then we can just do something like `<a http-header>Clear-Site-Data</a>`. But no need to block on that.